ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/ios-host.js

```diff
@@ -45,8 +45,8 @@ function init() {
     kmw.init({'app':device,'fonts':'fonts/'});//.then(function() {
         kmw['util']['setOption']('attachType','manual');
         kmw['oninserttext'] = insertText;
-        kmw['showKeyboardList'] = menuKeyDown;
-        kmw['hideKeyboard'] = hideKeyboard;
+        window['showKeyboardList'] = menuKeyDown;
+        window['hideKeyboard'] = hideKeyboard;
```

Comment:
I don't understand this change -- why are we moving these onto the global object?

Reply:
Found a simple way to move 'em back.